provider/googleProvider.js

```diff
@@ -41,6 +42,29 @@ class GoogleProvider {
       logging: google.logging('v2'),
       cloudfunctions: google.cloudfunctions('v1'),
     };
+
+    this.variableResolvers = {
+      gs: this.getGsValue,
+    };
+  }
+
+  getGsValue(variableString) {
+    const groups = variableString.split(':')[1].split('/');
+    const bucket = groups.shift();
+    const object = groups.join('/');
+
+    return this.serverless
+      .getProvider('google')
+      .request('storage', 'objects', 'get', {
+        bucket,
+        object,
+        alt: 'media',
+      })
+      .then(response => BbPromise.resolve(response))
+      .catch(err => {
+        const errorMessage = `Error getting value for ${variableString}. ${err.message}`;
+        return BbPromise.reject(new this.serverless.classes.Error(errorMessage));
```

Comment:
Let's just throw error, it's more natural, and effect will be same

Reply:
On my way!